src/redux/categories/categories.js

```diff
@@ -0,0 +1,18 @@
+const CHECK_STATUS = 'CHECK_STATUS';
+
+// eslint-disable-next-line default-param-last
```

Comment:
- [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿